core/mempool/src/lib.rs

```diff
@@ -153,7 +174,13 @@ impl Pool {
     pub fn snapshot_payload(&self) -> CryptoHash {
         // Put tx and receipts into an snapshot without erasing them.
         let transactions: Vec<_> =
-            self.transactions.write().expect(POISONED_LOCK_ERR).iter().cloned().collect();
+            self.transactions
+                .write()
+                .expect(POISONED_LOCK_ERR)
+                .iter()
```

Comment:
`iter()`->`values()`. Also might not need to specify `::<Vec<_>, _>` after that.

Reply:
Fixed. The type annotation is still necessary.